### PR TITLE
Improvements to He approximation.

### PR DESCRIPTION
**Pull request by aoifeboyle**
Much more efficient I think.

### CODE REVIEW

**Comment by aoifeboyle** [comment]
@wkerzendorf Merge if happy.

File: tardis/plasma/properties/nlte.py [hunk 1]
```diff
@@ -71,72 +70,36 @@ def calculate(self, level_boltzmann_factor, electron_densities,
...
         """
-        (partition_function_index, ionization_data_index, partition_function,
-            ionization_data) = HeliumNLTE.filter_with_helium_index(2, 1,
-            partition_function, ionization_data)
-        phis = (1 / PhiSahaLTE.calculate(g_electron, beta_rad,
-            partition_function, ionization_data)) * electron_densities * \
-            (1.0/g.ix[2,1,0]) * (1/w) * (t_rad/t_electron)**(0.5)
         return level_boltzmann_factor.ix[2].ix[0].mul(
-            pd.DataFrame(phis.ix[2].ix[1].values)[0].transpose())
+            g.ix[2].ix[0], axis=0) * (1./(2*g.ix[2].ix[1].ix[0])) * \
```
Comment (wkerzendorf) [inline tardis/plasma/properties/nlte.py]: you are not using the `multiindex` -> `.ix[2, 0]` for example.

**Comment by aoifeboyle** [comment]
@wkerzendorf Better? :)

**Comment by wkerzendorf** [comment]
Looks good. I guess tests already exist. I think this is mostly in your area of expertise, so merge when you think it's ready.

**Comment by aoifeboyle** [comment]
Yeah it still passes the appropriate module test.

**Pull request merged**